StudCity.Frontend/src/components/AllGroupsUser/UserGroup/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+export default function UserGroup({ group }) {
+	const baseLinkStyles = 'w-full mx-auto h-24 cursor-pointer flex flex-col hover:bg-customGreen rounded-md pl-3'
+	return (
+		<div className="">
+			<NavLink to={`/groups/${group.id}`} className={({ isActive }) => { return isActive ? `${baseLinkStyles} bg-customGreen` : baseLinkStyles }}>
```

Comment:
simply write: className={`${baseLinkStyles} ${isActive ? 'bg-customGreen' : ''}`}

Reply:
@Serhiy-dotcom I think you are confusing something, this is how active class names for `NavLink` are written.
But we can write something like that

`className={({ isActive }) => isActive ? `${baseLinkStyles} bg-customGreen` : baseLinkStyles }`

Reply:
@maryanchuk04 I'm pretty sure our codes doing the same thing, the only difference for active NavLink is that it has className bg-customGreen and not active NavLink doesn't have this class

Reply:
@Serhiy-dotcom I think your case not working because in your code isActive is not defined, but case @maryanchuk04  almost correct 
and possible correct case className={({ isActive }) => (isActive ? `${baseLinkStyles} bg-customGreen` : baseLinkStyles)}